packages/plugin-angular/package.json

```diff
@@ -25,7 +25,6 @@
     "build": "npm run clean && npm run build:esm2015 && npm run build:esm5",
     "build:esm2015": "ngc -p tsconfig.json",
     "build:esm5": "ngc -p tsconfig.esm5.json",
-    "test": "npm run build",
```

Comment:
I'm assuming we don't need to run this on test but I could be wrong. It can be reinstated along with the related package.json change.

Reply:
This "tests" the package by checking it compiles. We could do `tsc --noEmit` instead to make it more explicit?

Reply:
I've changed the script name from 'test' to 'test:types' so it gets run with the other type tests